### ISSUE TITLE
[Bug] Chapter download and read stuck

### ISSUE THREAD

**Issue by Lustsifer** [LagradOst/QuickNovel#27]
I can't download and read some chapter of a certain novel but that chapter clearly available when I open it using browser. It made me stuck at that chapter and cannot proceed to the next chapters that's why I make the previous issue #26 so hopefully I can continue reading.

![Screenshot_20210918-111247](https://user-images.githubusercontent.com/84452966/133872171-4ec16932-0571-4177-901c-9d0e5af9a5ff.png)
_https://wuxiaworld.site/novel/long-live-summons/_

![Screenshot_20210918-111535](https://user-images.githubusercontent.com/84452966/133872173-2c312091-147b-48cb-9706-56280a1bebf4.png)
_https://wuxiaworld.site/novel/oukoku-e-tsuzuku-michi/_

**Comment by LagradOst** [comment]
Is it working in the stream read? (I dont know if my parser stopped or if the chapter contains some invalid html)

**Comment by Lustsifer** [comment]
Stream read isn't working too, it just show _error loading chapter_ and keep loading.
![Screenshot_20210919-041928](https://user-images.githubusercontent.com/84452966/133908793-9af532ba-61e2-43cf-a733-47ec48e85b48.png)

**Comment by Lustsifer** [comment]
This bug seems to be the same with issue #25 because I also met some novel that I can't open. Although the site is different but wuxiaworld and boxnovel usually has the same problem.

**Comment by LagradOst** [comment]
Yes, it is the same issue. It seams like the provider gives an invalid html, so it tried again and again to download that chapter.
